extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -75,9 +75,13 @@ export const URL_EXPERIMENT_MAPPING = {
 /**
  * @param {!Window} win
  * @param {!Element} element
+ * @param {!boolean} useRemoteHtml
```

Comment:
Nit: No need for the ! on primitive types (here and elsewhere in PR).

Reply:
huh? Why don't primitive types need ! when relevant in the jsdoc?